tests/docker/package_managers/BUILD

```diff
@@ -0,0 +1,134 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
We also need these case https://github.com/GoogleContainerTools/base-images-docker/blob/master/BUILD#L10 which are located at the root of the repository.

Reply:
added to testing/download_pkgs_at_root